types/three/examples/jsm/renderers/common/Renderer.d.ts

```diff
@@ -36,7 +92,121 @@ export default class Renderer {
 
     setAnimationLoop(callback: ((time: DOMHighResTimeStamp) => void) | null): Promise<void>;
 
+    /**
+     * Sets the pixel ratio of the Renderer.
+     */
     setPixelRatio(value?: number): void;
 
+    /**
+     * Sets the width and height of the output canvas and optionally updates the CSS style on the DOM element.
+     */
     setSize(width: number, height: number, updateStyle?: boolean): void;
+
+    /**
+     * Gets the Rendering Context of the renderer.
+     */
+    getContext(): RenderingContext;
+
+    /**
+     * Gets the current pixel ratio of the renderer.
+     */
+    getPixelRatio(): number;
+
+    getDrawingBufferSize(target: Vector2): Vector2;
+
+    getSize(target: Vector2): Vector2;
+
+    setDrawingBufferSize(width: number, height: number, pixelRatio: number): void;
+
+    /**
+     * Sets the custom opaque sort function for the RenderLists. Pass null to use the default painterSortStable function.
+     */
+    setOpaqueSort(method: (a: any, b: any) => number): void;
+
+    /**
+     * Sets the custom transparent sort function for the RenderLists. Pass null to use the default reversePainterSortStable function.
+     */
+    setTransparentSort(method: (a: any, b: any) => number): void;
```

Comment:
Can we default to `unknown` instead of `any`?

Reply:
yep, I can amend that no problem